docs/commands.md

```diff
@@ -341,7 +518,25 @@ This command is extremely forgiving and using it with out of range indexes will
 
```

Comment:
```suggestion
This command is extremely forgiving and using it with out of range indexes will not produce an error. If `start` is larger than the array's size or `start` > `stop`, the return value will be 0 and the resulting array will be empty. If `start` is < 0 then it is interpreted from the end. If `stop` is larger than the end of the array, it will be treated like the last element in it.
```